lib/shrine/plugins/determine_mime_type.rb

```diff
@@ -13,15 +13,21 @@ def self.configure(uploader, opts = {})
         end
 
         uploader.opts[:mime_type_analyzer] = opts.fetch(:analyzer, uploader.opts.fetch(:mime_type_analyzer, :file))
+        uploader.opts[:analyzer_options] = opts.fetch(:options, uploader.opts.fetch(:analyzer_options, {}))
```

Comment:
Let's name the saved option `:mime_type_analyzer_options`, because `store_dimensions` plugin also has the concept of "analyzers", so if we decide to allow passing options there, then if we'd name it `:analyzer_options` as well, it would clash.